src/plone/api/env.py

```diff
@@ -201,6 +202,17 @@ def test_mode():
     return IS_TEST
 
 
+def read_only_mode():
+    """Returns True if you are running the zope instance on an read only ZODB.
+
+    :Example: :ref:`env_read_only_mode_example`
+    """
+    conn = Globals.DB.open()
```

Comment:
Do we need to open a new db connection here? Or is it probably better to get the current connection or one from the pool if there is no current one?